bread/utils/queryset_from_fields.py

```diff
@@ -78,6 +64,8 @@ def get_field_queryset(fields, queries, prefix="", **kwargs):
                 foreign_fields = foreignkey_field._meta.fields
 
             new_prefix = prefix + "__".join([foreignkey_field.name, ""])
-            qs |= get_field_queryset(foreign_fields, queries, new_prefix, **kwargs)
+            qs |= get_field_queryset(
+                foreign_fields, searchquery, new_prefix, follow_relationships - 1
+            )
 
     return qs
```

Comment:
This implementation seems to work pretty good. I adjusted the recursion a bit (so we could also go deeper if necessary) and I wanted the query to be a single string, not a list. More complicated queries should be done qith djangoql.